### PR TITLE
Replace SkipBytes and SeekOffset with Offset setter

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@nguerrera

File: src/System.Reflection.Metadata/tests/Utilities/BlobReaderTests.cs [hunk 1]
```diff
@@ -266,35 +349,35 @@ public unsafe void ReadFromMemoryBlock()
...
                 Assert.Throws<BadImageFormatException>(() => block.GetMemoryBlockAt(0, -1));
                 Assert.Throws<BadImageFormatException>(() => block.GetMemoryBlockAt(-1, 0));
-                Assert.Throws<BadImageFormatException>(() => block.GetMemoryBlockAt(-Int32.MaxValue, Int32.MaxValue));
-                Assert.Throws<BadImageFormatException>(() => block.GetMemoryBlockAt(Int32.MaxValue, -Int32.MaxValue));
-                Assert.Throws<BadImageFormatException>(() => block.GetMemoryBlockAt(Int32.MaxValue, Int32.MaxValue));
-                Assert.Throws<BadImageFormatException>(() => block.GetMemoryBlockAt(block.Length, -1));
+                Assert.Throws<BadImageFormatException>(() => block.GetMemoryBlockAt(-int.MaxValue, int.MaxValue));
+                Assert.Throws<BadImageFormatException>(() => block.GetMemoryBlockAt(int.MaxValue, -int.MaxValue));
+                Assert.Throws<BadImageFormatException>(() => block.GetMemoryBlockAt(int.MaxValue, int.MaxValue));
+                Assert.Throws<BadImageFormatException>(() => block.GetMemoryBlockAt(block.  Length, -1));
```
Comment (nguerrera) [inline src/System.Reflection.Metadata/tests/Utilities/BlobReaderTests.cs]: nit: extra space.

**Comment by nguerrera** [comment]
LGTM

File: src/System.Reflection.Metadata/tests/Utilities/BlobReaderTests.cs (hunk 1 above)
Comment (tmat) [inline src/System.Reflection.Metadata/tests/Utilities/BlobReaderTests.cs]: Fixed.

**Comment by tmat** [comment]
test  Innerloop CentOS7.1 Release Build and Test  please

**Pull request merged**